.github/workflows/ci.yml

```diff
@@ -106,6 +106,12 @@ jobs:
       - name: Get CPU info
         id: cpu_info
         run: echo "cpu_cores=$(nproc)" >> $GITHUB_ENV
+      - name: Install chromedriver
+        uses: nanasess/setup-chromedriver@v2
+      - run: |
+          export DISPLAY=:99
```

Comment:
I don't think we need this, we can run it in headless mode, right? 

You might want to check how Rails will do it for new apps here https://github.com/rails/rails/pull/50508/files#diff-dc7109b564892719463be7ade1c27d742684530b8235295c6ed32cb579495712

Reply:
:+1: